src/libponyrt/gc/cycle.c

```diff
@@ -139,6 +139,9 @@ DEFINE_HASHMAP(ponyint_perceivedmap, perceivedmap_t, perceived_t,
   perceived_hash, perceived_cmp, ponyint_pool_alloc_size,
   ponyint_pool_free_size, perceived_free);
 
+DECLARE_STACK(ponyint_actorstack, actorstack_t, pony_actor_t);
+DEFINE_STACK(ponyint_actorstack, actorstack_t, pony_actor_t);
```

Comment:
I don't see what could be a better name. Maybe we could move the stack declaration closer to where it is used instead?